src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.product-card {
+  width: $-width-card;
+
+  border: 1px solid $-c-ligthGray;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+
+    width: $-width-image;
+    height: $-heigth-image;
+
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    @extend %text-medium;
+    line-height: 18px;
+    color: $-c-darkBlue;
+
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    @extend %text-small;
+
+    color: $-c-gray;
+
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    @extend %space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+  }
+
+  &__rewievs {
+    display: flex;
+    align-items: center;
+
+    @extend %text-small;
+    color: $-c-darkBlue;
+  }
+
+  &__price {
+    @extend %space-between;
+
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    @extend %text-small;
+    font-size: 12px;
+    line-height: 18px;
+    color: $-c-gray;
+  }
+
+  &__number {
+    @extend %text-large;
+    font-size: 16px;
+    line-height: 18px;
+    color: $-c-darkBlue;
+  }
```

Comment:
maybe it's better to delete font-weight from extends, because you use this element but change 2 of 3 definition after.